transcom/mymove
# INT B-18910 gun safe frontend

## [B-18910](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18910)

## Summary

This branch adds the frontend changes necessary to allow an SC or TOO to indicate whether or not a customer is entitled to move a gun safe as part of their move.

1. There is now a checkbox on the "Edit Allowances" page with the label "Gun safe authorized" which allows the govt. user to indicate the gun safe status.
2. On the shipment card that is under the "Edit Allowances" link, a row has been added that indicates whether a gun safe has been authorized or not. This allows the user to easily see that their changes have been saved.

### How to test

1. Access the Move Details page of any move as an SC
2. Scroll down to the allowances card, verify that there is a new row at the bottom for gun safe, and that it is "Unauthorized".
3. Click the link, check the box for "Gun safe authorized", then click submit.
4. Scroll back down to the "Allowances" card and verify that the Gun Safe value is now "Authorized".
5. Repeat steps 1-4 while logged in as a TOO.

The new row in the shipment card:
![Screenshot 2024-04-10 at 10 07 23 AM](https://github.com/transcom/mymove/assets/146959895/9900cb47-21cd-4460-9f30-48a1a0d8755d)

The new checkbox on the Edit Allowances page:
![Screenshot 2024-04-10 at 10 08 06 AM](https://github.com/transcom/mymove/assets/146959895/bccc8407-8fa0-4f73-92b1-2ef0a19fd9fa)
